src/app/loadout-builder/loadout-builder-reducer.ts

```diff
@@ -360,10 +360,16 @@ function lbConfigReducer(defs: D2ManifestDefinitions) {
         };
       }
       case 'statConstraintChanged': {
-        const { constraint } = action;
-        const newStatConstraints = state.resolvedStatConstraints.map((c) =>
-          c.statHash === constraint.statHash ? constraint : c,
-        );
+        const { constraint: newConstraint } = action;
+        const newStatConstraints = state.resolvedStatConstraints.map((c) => {
+          if (c.statHash === newConstraint.statHash) {
+            // Previously ignored stats' maxes need to be bumped up to T10 again
+            const maxTier = newConstraint.ignored ? 0 : c.ignored ? 10 : c.maxTier;
+            return { ...newConstraint, maxTier };
+          } else {
+            return c;
+          }
+        });
```

Comment:
This is a bit awkward - maybe this reducer action needs to be split between toggle enabled-ness and updating the constraint values.